tests/D2L.CodeStyle.Analyzers.Test/Async/Generator/AsyncToSyncMethodTransformerTests.cs

```diff
@@ -38,6 +38,70 @@ internal sealed class AsyncToSyncMethodTransformerTests {
 		Assert.AreEqual( "[Blocking] int Hello() => 4;", actual.Value.ToFullString() );
 	}
 
+	[Test]
+	public void Silly() {
+		var actual = Transform( @"[GenerateSync]
+async Task<int> HelloAsync( int[] q ) {
+	if( 6 == (7 - 1)*2 ) {
+		return sizeof( ""hello"" );
+	} else if( q[0]++ == ++q[1] ) {

```

Comment:
Hm, this actually shouldn't be passing because the identifier `q` isn't handled... looking into it

Reply:
Fixed -- I had forgotten another `Transform`.